packages/components/src/Toast/Toast.module.css

```diff
@@ -2,6 +2,8 @@
   @apply w-full max-w-100 pl-4 pr-2 grid gap-2 items-center;
   @apply border border-l-4 rounded shadow-1;
 
+  border-color: var(--banner-border-light-color);
+  border-left-color: var(--banner-border-dark-color);
```

Comment:
The shared styles are setting variables, not actual border colors, because  the `Banner` border are different on desktop and mobile, so I had to update the `Toast` to use the variables.